src/components/FreePlugins.js

```diff
@@ -1,18 +1,27 @@
+import { __ } from "@wordpress/i18n";
 import { Spinner } from "@yoast/ui-library";
 import { FreePluginsDefinition } from "../configs/FreePlugins.config";
+import { RuntimeSdk } from "../sdk/runtime";
 import { Section } from "./Section";
 import { useCardManager } from "./useCardManager";
 
+const Text = RuntimeSdk.hasCapability("isEcommerce")
+  ? __(
+      "Improve your store with these free add-ons included in your plan.",
```

Comment:
Can we use template literals here ?

Reply:
The only concern is with translation strings. Maybe we should look into `sprintf` or something similar.